addon/components/rdf-input-fields/concept-scheme-selector.js

```diff
@@ -24,13 +24,21 @@ export default class RdfInputFieldsConceptSchemeSelectorComponent extends InputF
 
   constructor() {
     super(...arguments);
+
     this.loadOptions();
-    this.loadProvidedValue();
+    // this.loadProvidedValue();
```

Comment:
This shouldn't have been committed I think?

```suggestion
    this.loadProvidedValue();
```